src/sql/run/resultset.py

```diff
@@ -504,10 +505,11 @@ def _nonbreaking_spaces(match_obj):
 
 
 def _statement_is_select(statement):
-    statement_ = statement.lower().strip()
-    # duckdb also allows FROM without SELECT
-    return (
-        statement_.startswith("select")
-        or statement_.startswith("from")
-        or statement_.startswith("with")
-    )
+    if statement and isinstance(statement, str):
```

Comment:
why do we need to check the type?

Reply:
My bad I was making the function more flexible, but we don't need to check since we are already checking that [removed!](https://github.com/ploomber/jupysql/pull/797/commits/2cfaac6230cae0cffead4ee3a6186d04a079fdad#diff-ee061806ed7a4f5a79d8571f64241031f5c61240dbe59e548de0666a7996ab88L508)